docs/installation-guide.md

```diff
@@ -0,0 +1,52 @@
+# Get Started with Lily
+
+Here is an in depth guide on how to set up Lily for your own personal use. If you have issues with this guide, please join [Iris's discord](https://discord.gg/jQJnav2jPu) for support.
+
+### Step 1 - Set up a bot using the discord developer portal
+
+Head over to the [Discord Developer Portal](https://discord.com/developers/applications) and press the "New Application" button in the top right corner.
+
+Give it a name and press create. Then, head to the bot tab on the left and click "Add Bot". Click "Yes, do it!" Once you have your bot, scroll down and make sure that the "PRESENCE INTENT", "SERVER MEMBERS INTENT", and "MESSAGE CONTENT INTENT" switches are all toggled to on. Once that's done, go to the OAuth2 tab on the left sidebar, scroll down to the checkboxes, and select the ones for "bot" and "applications.commands". Copy the URL, enter it into your browser, select the server you want to add your Lily to, and click authorize. If everything worked correctly, your Lily instance should appear in the members list of that server.
+
+### Step 2 - Clone the repository
+
+To clone the Lily repository, open the command line for your respective OS and run `git clone https://github.com/IrisShaders/LilyBot.git`
+This will clone Lily to your home directory.
+
+### Step 3 - Set up your .env file
+
+In the root directory of the files you've cloned (It has files such as `README.md` and `CONTRIBUTING.md`), create a file named .env. You should fill your file using the format below with the relevant details filled in.
+
+```
+TOKEN= TOKEN_YEEF
```

Comment:
This format is technically incorrect, and I had many issues on Kord. The space after the equals cannot exist

Reply:
Should be better now